packer/ansible/roles/cyhy_commander/tasks/main.yml

```diff
@@ -2,8 +2,13 @@
 # tasks file for cyhy_commander
 
 #
-# Install unzip; it is needed by cyhy_core/var/load_places.sh
+# Install curl and unzip; it is needed by cyhy_core/var/load_places.sh
 #
+- name: Install curl
+  apt:
+    name: curl
```

Comment:
Name can be a list, like this:
```
name:
  - curl
  - unzip
```

So you could combine both installs into a single Ansible command.

If you do make this change, can you also change "it is" to "they are" in the comment?

Reply:
Good catches, thanks @jsf9k!